python-discord/site
# Dewikification: Migrate site content to markdown.

This PR migrates pages under https://pythondiscord.com/pages/ to the site repo as part of dewikification.

### How to Review
1. Run the branch locally.
2. Compare each page with the corresponding page on the current website.
3. Don't suggest grammar/phrasing changes for any existing page content, however new content like category and page descriptions can be brought up for improvement.
4. Check if links function as intended, and that any images used are local static images and not hotlinked to another website.
5. Get a thank you from me for taking the time to review.

**Notes**
- Certain files have trailing spaces; If the spaces are 2-wide, this is intentional for proper spacing on the displayed markdown. 1-wide trailing spaces can be cleaned up.

**Fixes**
- 10e1727c703d3cb91321e9ef51f54a0117efed19 Removed duplicates from the `Sub-Articles` button in category-page hybrids.
